app/Models/Setting.php

```diff
@@ -13,4 +13,15 @@ class Setting extends Model
         'key',
         'value'
     ];
+
+
+    //ASK SAM IF YOU WANT TO EAGER LOAD THE USER WITH THE SETTING? Could be useful.
+    /*protected $with = [
+        'user'
+    ]; */
```

Comment:
While this is very useful to have, we tend not to eager load all the time unless the model always needs it. 

Usually in our routes, we would eager load on the fly like this:

```php
$model = Settings::with('user')->get();
```